static/js/reducers/ui.js

```diff
@@ -25,8 +26,24 @@ import {
 } from '../actions/ui';
 import { HIGH_SCHOOL, ASSOCIATE, BACHELORS, MASTERS, DOCTORATE } from '../constants';
 import { calculateDegreeInclusions } from '../util/util';
+import type { Action } from '../flow/generalTypes';
 
-export const INITIAL_UI_STATE = {
+export type UIState = {
```

Comment:
yeah, I was hoping for something that could just take an object literal and return a named type from it, but it doesn't seem like that's supported yet.